src/PrettierEditProvider.ts

```diff
@@ -65,7 +56,18 @@ function format(text: string, path: string): string {
         parser: parser,
         semi: config.semi,
         useTabs: config.useTabs,
-    });
+    };
+    if (config.eslintIntegration) {
+        const prettierEslint = requireLocalPkg(path, 'prettier-eslint') as PrettierEslintFormat;
```

Comment:
Is this really useful ?
prettier-eslint already loads what's in the workspace. And thus we could avoid some troubles with people having an older version (< 6.1.0) without `fallbackPrettierOptions`